### PR TITLE
Adapt documentation for trigger sequence

### PR DESCRIPTION
**Pull request by laneli**
Signed-off-by: Elisabeth Lang <elisabeth.lang@dynatrace.com>

### CODE REVIEW

File: content/docs/0.14.x/bridge/manage_sequences/index.md [hunk 1]
```diff
@@ -65,13 +65,20 @@ Fields:
...
+{{< popup_image
+link="./assets/trigger-08-failed-sequence-dt-service"
+caption="Dynatrace-service will fail sequence if end date is too far in the future">}}
 
+#### **Start / end date**
 Mandatory fields:
 
-* Start at - Defines the start date and time from when the evaluation is performed. Has to be **after** the end date.
-* End at - Defines the end date and time until when the evaluation is performed. Has to be **before** the start date.
+* Start at - Defines the start date and time from when the evaluation is performed. Has to be before the end date.
```
Comment (oleg-nenashev) [inline content/docs/0.14.x/bridge/manage_sequences/index.md:80]: oops

**Review by oleg-nenashev — APPROVED** [review]
LGTM. Would appreciate reviews from Keptn core code reviewers

File: content/docs/0.14.x/bridge/manage_sequences/index.md [hunk 2]
```diff
@@ -65,13 +65,20 @@ Fields:
 link="./assets/trigger-05-trigger-evaluation-timeframe"
 caption="Sequences">}}
 
-#### **Start / end date**
-Please mind, that an evaluation is performed in the past. That means that the start date has to be **after** the end date.
+Special considerations:
+* If you set a timeframe but no start date, the evaluation will be performed to the past for the given timeframe. This is the only case where evaluations will be performed to the past.
```
Comment (StackScribe) [inline content/docs/0.14.x/bridge/manage_sequences/index.md:69]: If you set a timeframe but no start date, the evaluation is performed on the previous timeframe span.  This is the only case where evaluations are performed on data from the past.

File: content/docs/0.14.x/bridge/manage_sequences/index.md [hunk 3]
```diff
@@ -65,13 +65,20 @@ Fields:
...
+* As soon as a start date is given, the evaluation is always performed from this point to an end date. The end date is either given or will be calculated from the timeframe.
```
Comment (StackScribe) [inline content/docs/0.14.x/bridge/manage_sequences/index.md:70]: When a start date is specified, the evaluation is perfromed from this point to an end date.  The end date can be specified or is calculated from the timeframe value.

File: content/docs/0.15.x/bridge/manage_sequences/index.md [hunk 4]
```diff
@@ -65,13 +65,20 @@ Fields:
 link="./assets/trigger-05-trigger-evaluation-timeframe"
 caption="Sequences">}}
 
-#### **Start / end date**
-Please mind, that an evaluation is performed in the past. That means that the start date has to be **after** the end date.
+Special considerations:
+* If you set a timeframe but no start date, the evaluation will be performed to the past for the given timeframe. This is the only case where evaluations will be performed to the past.
```
Comment (StackScribe) [inline content/docs/0.15.x/bridge/manage_sequences/index.md:69]: same as above

**Review by StackScribe — APPROVED** [review]
Nicely done!  I made a couple editorial suggestions -- use or discard as you like.

**Comment by laneli** [comment]
@StackScribe can I ask you to have another look 🙂

File: content/docs/0.14.x/bridge/manage_sequences/index.md [hunk 5]
```diff
@@ -65,13 +65,20 @@ Fields:
...
 caption="Sequences">}}
 
-#### **Start / end date**
-Please mind, that an evaluation is performed in the past. That means that the start date has to be **after** the end date.
+Special considerations:
+* If you set a timeframe but no start date, the evaluation is performed on the previous timeframe span. This is the only case where evaluations are performed on data from the past.
+* When a start date is specified, the evaluation is performed from this point to an end date. The end date can be specified or is calculated from the timeframe value.
+* Please note: If the timeframe is calculated to an end date too far into the future, evaluation sequences might fail depending on the configured SLI provider.
+
+{{< popup_image
```
Comment (johannes-b) [inline content/docs/0.14.x/bridge/manage_sequences/index.md:73]: The images are not displayed in the preview, please double-check: 

image

**Pull request merged**